examples/tokenize/tokenize.cpp

```diff
@@ -3,40 +3,391 @@
 
 #include <cmath>
 #include <cstdio>
+#include <iostream>
```

Comment:
`<iostream>` should not be needed

Reply:
removed and then ran

cmake -B build
cmake --build build --config Release

and confirmed no compile issue.